pkg/sentry/platform/ring0/entry_arm64.s

```diff
@@ -540,6 +544,10 @@ TEXT ·Start(SB),NOSPLIT,$0
 	ORR $0xffff000000000000, RSV_REG, RSV_REG
 	WORD $0xd518d092        //MSR R18, TPIDR_EL1
 
+	// save sentry's tls to memory.
+	MRS TPIDR_EL0, R1
+	MOVD R1, CPU_REGISTERS+PTRACE_TLS(RSV_REG)
```

Comment:
In my opinion, sentry's tls has already been saved into memory in bluepillArchEnter().
So, this code is redundant

Reply:
Yes, you are right. bluepillArchEnter() has already save the TLS